packages/react-components/react-popover/src/components/PopoverSurface/usePopoverSurfaceStyles.styles.ts

```diff
@@ -7,6 +7,7 @@ import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const popoverSurfaceClassNames: SlotClassNames<PopoverSurfaceSlots> = {
   root: 'fui-PopoverSurface',
+  arrow: 'fui-PopoverSurface__arrow',
```

Comment:
Out of curiosity, is this the naming convention used in FUIR? I didn't see anything about it here: https://github.com/microsoft/fluentui/wiki/Coding-Style

`fui-[component]__[slot]`

Reply:
Ya this is our default naming schema, would be good to add here